.semaphore/setup.sh

```diff
@@ -1,8 +1,8 @@
 cache restore $SEMAPHORE_PROJECT_NAME-dep-$(checksum .semaphore/setup.sh)
 # checks if the packages are already installed
-if [ ! -d '/packages' -o ! -f '/packages/go1.13.6.linux-amd64.tar.gz' ]; then
+if [ ! -d '/packages' -o ! -f '/packages/go1.13.8.linux-amd64.tar.gz' ]; then
```

Comment:
One thing I don't understand is why there is a go installation in setup. I thought semaphore/golang should provide it already. Do you know why it was done this way?

Reply:
Yeah, I think just using the appropriate golang image should be sufficient.
@kianjones4 any idea why we need to run `setup.sh` at all?

Reply:
@eytan-avisror setup.sh also caches kube-builder.

Reply:
Cleaning this up can be a separate PR though.

Reply:
we don't really need kubebuilder in this context - only controller-gen.
Perhaps setup.sh can be removed in future PRs all together